packages/flutter/lib/src/services/activity.dart

```diff
@@ -60,7 +60,7 @@ void updateTaskDescription(String label, Color color) {
 
   TaskDescription description = new TaskDescription()
     ..label = label
-    ..primaryColor = color?.value;
+    ..primaryColor = color?.value ?? 0;
```

Comment:
0 is a pretty terrible default (I think it's transparent? What does Android even do with that).

I think if we want to require a colour we should just assert when no colour is given. Since color is a required argument here, asserting non-null seems legit.

Can android be told to use a default value? If so maybe a better solution is to support that (e.g. by making it nullable).